Travel/Travel/Models/Users.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Travel.Models
+{
+    public class Users
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public List<Trips> Trips { get; set; }
```

Comment:
Use interfaces instead of implementation as types. Read about the difference in IList<T>, ICollection<T>, and IEnumerable<T> and which one will be most suitable for Entity Framework